src/platform/packages/shared/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -44,6 +44,7 @@ export const getDocLinks = ({ kibanaBranch, buildFlavor }: GetDocLinkOptions): D
       ? `${ELASTIC_DOCS}deploy-manage/deploy/elastic-cloud/serverless`
       : `${ELASTIC_DOCS}get-started`,
     apiReference: `${API_DOCS}`,
+    serverlessReleaseNotes: `${ELASTIC_DOCS}release-notes/cloud-serverless`,
```

Comment:
an alternative option would be to make this `latestReleaseNotes` and set the value based on `isServerless` then always add this link to the help menu.